cob_collision_velocity_filter/include/cob_collision_velocity_filter.h

```diff
@@ -51,7 +51,12 @@
 #include "velocity_limited_marker.h"
 
 // Costmap for obstacle detection
+#if ROS_VERSION_MINIMUM(1, 14, 0)
+#include <tf2_ros/transform_listener.h>	
```

Comment:
we still would need a `<build_depend>tf2_ros</build_depend>` in the `package.xml`, right?

Reply:
see 8e1efb2

Reply:
You could move the include to the cpp. It's just used there.